java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -91,4 +89,118 @@ protected static ConnectionManager buildConnectionManager(ChannelHandler channel
     protected static CommandManager buildCommandManager(ChannelHandler channelHandler) {
         return new CommandManager(channelHandler);
     }
+
+    /**
+     * Extracts the response from the Protobuf response and either throws an exception or returns the
```

Comment:
if throws add `@throws` or update the doc

Reply:
You can also add `throws` to function signature. Since our all exceptions are unchecked, this won't require code change.

Reply:
It's fine as is.